47.md

```diff
@@ -0,0 +1,112 @@
+NIP-47
+======
+
+Nostr Wallet Connect
+--------------------
+
+`draft` `optional` `author:kiwiidb` `author:bumi` `author:semisol` `author:vitorpamplona`
+
+## Rationale
+
+Paying zaps should be possible without the user needing to open a different app to only pay a Lightning invoice.
+This NIP describes a way for users to control a remote Lightning node or a custodial Lightning wallet. When self-hosting, this setup does not require the user to run their own server, thereby bypassing certain hurdles that are commonly encountered when trying to connect to a Lightning node remotely.
+
```

Comment:
This NIP is designed to forward the responsibility of executing lightning payments to specialized Nostr micro apps whose sole purpose is to connect with the user's desired lightning service, generally using the service's proprietary API, and establish security controls to approve such transactions. Regular Nostr Clients are then exempt from implementing payment interfaces with each lightning provider out there and from implementing security settings to authorize certain payment types.